core-bundle/src/Resources/contao/templates/frontend/fe_page.html5

```diff
@@ -94,5 +94,8 @@
 
   <?= $this->mootools ?>
 
+  <?php if ($this->checkCookiesUrl): ?>
+    <img src="<?= $this->checkCookiesUrl ?>" width="1" height="1" class="invisible" alt role="presentation">
```

Comment:
Why did you add the `role="presentation"` attribute here?

Reply:
For screen readers. I read up on the topic and apparently most of them ignore images that have an `alt` attribute set to `null` but I figured we can just be very sure and add `role` too :)

Reply:
So why not `aria-hidden`?

Reply:
Hmm according to https://timwright.org/blog/2016/11/19/difference-rolepresentation-aria-hiddentrue/ it would actually be even better. Changed in 8ffc3194d8949a1240bd5aeb3cc98f1928688abb